keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/BakeryLifecycleMonitor.kt

```diff
@@ -117,7 +117,7 @@ class BakeryLifecycleMonitor(
         vmType = detail.vmType,
         cloudProvider = detail.cloudProviderType,
         appVersion = detail.`package`.substringBefore("_all.deb").replaceFirst("_", "-"),
-        baseAmiVersion = imageService.findBaseAmiVersion(detail.baseAmiId, detail.region),
+        baseAmiName = imageService.findBaseAmiName(detail.baseAmiId, detail.region),
```

Comment:
@emjburns I did change that bit

Reply:
oh cool I'm blind. thanks for pointing it out!